package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-check-updates",
-  "version": "4.0.1",
+  "version": "4.1.0",
```

Comment:
Changing the package version is up to the package maintainers to decide and should never be done in a PR unless requested.